lib/cc/analyzer/formatters/plain_text_formatter.rb

```diff
@@ -14,8 +14,8 @@ def started
         def write(data)
           if data.present?
             json = JSON.parse(data)
-            if @active_engine
-              json["engine_name"] = @active_engine.name
+            if current_engine
```

Comment:
Went ahead with this anyway now that I refactored the spec such that it doesn't matter